test/runtests.jl

```diff
@@ -1,5 +1,5 @@
 using RHEOS
-using SpecialFunctions
+#using SpecialFunctions
```

Comment:
I think not exporting this will cause test failures?

Reply:
Well, apparently not. So it is possible we don't need at after all. @ab2425 do you know why we have this requirement in the tests?

I first commented the line to resolve a local conflict but then realised it did work fine as it is too.